llvm/lib/Target/AMDGPU/AMDGPUCodeGenPassBuilder.cpp

```diff
@@ -28,8 +36,51 @@ AMDGPUCodeGenPassBuilder::AMDGPUCodeGenPassBuilder(
 }
 
 void AMDGPUCodeGenPassBuilder::addPreISel(AddIRPass &addPass) const {
-  // TODO: Add passes pre instruction selection.
-  // Test only, convert to real IR passes in future.
+  const bool LateCFGStructurize = AMDGPUTargetMachine::EnableLateStructurizeCFG;
```

Comment:
Does this function run yet, or is this just preparatory work/NFC?

Reply:
It does run. We have a handful of tests running new PM already. I'm trying to avoid test failures caused by not running the same set of passes in the future